### DISCUSSION TITLE
Image URI Encode

### DISCUSSION THREAD

**Discussion by OskarEichler** [librespot-org/librespot-java#531] [General]
Hey @devgianlu,

Do you know how to convert the Spotify Image URL (https://i.scdn.co/image/ab6761610000e5eb69364c5c9d9d06d5bb3a2fbb) to an Image ID and vice versa? It seems like only a few characters change to determine the size of the image. We are interested in deconstructing the URL so that we can store the image ID and then can generate the URL depending on which image scale we want to serve to the user.

Please let me know how this is encoded. 

All the best,
Oskar

**Comment by devgianlu** [comment]
Can't really remember, but I think there's no relation between the CDN URL and the image ID.
